client/crypt.c

```diff
@@ -10,17 +10,14 @@
 #include <openssl/err.h>
 #include <openssl/rand.h>
 #include <openssl/ecdsa.h>
+#include <secp256k1.h>
```

Comment:
Please keep the former usage of secp256k1, since it's not working on Mac yet.

Reply:
Here I have tested and compiled with cmake, please see CMakeLists.txt
if (NOT DEFINED ENV {SECP256K1_ROOT_DIR})
  find_package (PkgConfig)
  pkg_search_module (SECP256K1 REQUIRED libsecp256k1)
endif ()

It is not recommended to continue to rely on xcode. There is a recursive stack overflow problem in xcode. Cmake is a cross-platform solution, but first you need to compile and install secp256k1.